Kernel/Arch/PageDirectory.h

```diff
@@ -1,156 +1,17 @@
 /*
- * Copyright (c) 2018-2021, Andreas Kling <kling@serenityos.org>
+ * Copyright (c) 2022, Timon Kruiper <timonkruiper@gmail.com>
```

Comment:
These are normally only additive, it's (IMHO) in bad taste to remove someone else's copyright like.

Reply:
Yeah true, the file basically completely changed, but anyway it's indeed better to keep the copyright. I pushed changes for this!